lib/index.d.ts

```diff
@@ -0,0 +1,241 @@
+import { AWS } from '@serverless/typescript';
+
+declare module '@serverless/typescript' {
+  interface AWS {
+    stepFunctions?: {
+      stateMachines: StateMachines;
+      validate?: boolean;
+    };
+  }
+}
+
+type StateMachines = {
+  [stateMachine: string]: {
+    id: string;
+    name: string;
+    definition: Definition;
+    tracingConfig?: TracingConfig;
```

Comment:
```suggestion
    tracingConfig?: TracingConfig;
    events?: any;
    dependsOn?: any;
```

Not really sure what would be their types.

Reply:
Looking at the source code:
https://github.com/serverless-operations/serverless-step-functions/blob/28fa07b254bd2f6fef934fe940911dd36fa77615/lib/deploy/stepFunctions/compileStateMachines.schema.js#L37-L40
https://github.com/serverless-operations/serverless-step-functions/blob/28fa07b254bd2f6fef934fe940911dd36fa77615/lib/deploy/stepFunctions/compileStateMachines.schema.js#L63

```suggestion
    tracingConfig?: TracingConfig;
    events?: any[];
    dependsOn?: string | string[];
```